src/components/Fields/Extensions.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+
+import { OptionsContext } from '../OptionsProvider';
+
+import { SchemaModel } from '../../services/models';
+import { FieldDetail } from './FieldDetail';
+
+export interface ExtensionsProps {
+  schema: SchemaModel;
+}
+
+export class Extensions extends React.PureComponent<ExtensionsProps> {
+  redocExtensions = [
+    'x-circular-ref',
+    'x-code-samples',
+    'x-displayName',
+    'x-examples',
+    'x-ignoredHeaderParameters',
+    'x-logo',
+    'x-nullable',
+    'x-servers',
+    'x-tagGroups',
+    'x-traitTag',
+  ];
+
+  render() {
+    const { schema } = this.props;
+    const fullSchema = schema.schema;
+    const extensionList = Object.keys(fullSchema).filter(
```

Comment:
Also, I think there is sense to do these filterings only when `options.showExtensions` is true.

I would add method called `getExtension`:

```ts
getExtensions() {
  const { schema } = this.props;
  const fullSchema = schema.schema;
  return Object.keys(fullSchema).filter(
    key => key.startsWith('x-') && !this.redocExtensions.includes(key),
  );
}
```

and then just used it in the `render` like bellow:

```ts
render() {
  return (
    <OptionsContext.Consumer>
      {options => (
        <>
          {options.showExtensions &&
            this.getExtensions().map(key => (
              <FieldDetail key={key} label={key} value={fullSchema[key]} />
            ))}
        </>
      )}
    </OptionsContext.Consumer>
  );
}
```